package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@tignum/eslint-config",
+  "version": "0.0.1",
+  "description": "eslint config based in eslint-config-airbnb",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/TIGNUM/eslint-config-tignum.git"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com"
+  },
+  "author": "m.rosero@tignum.com",
+  "license": "MIT",
+  "homepage": "https://github.com/TIGNUM/eslint-config-tignum#readme",
+  "devDependencies": {
+    // "eslint-plugin-sonarjs": "^0.13.0",
```

Comment:
This looks weird, does it work for you to have comments on a json file?

Reply:
You can enable it in vscod, it's just a reminder for the readme

Reply:
It is not only the editor. No package manager likes them
![image](https://user-images.githubusercontent.com/1942073/163274460-58e9c7b6-aa62-4efb-9dd0-e99eb703c1fd.png)
![image](https://user-images.githubusercontent.com/1942073/163274606-7d338cec-aa61-40f5-9416-c44dfeffbd52.png)
![image](https://user-images.githubusercontent.com/1942073/163274635-29fcd3af-2828-4273-8b57-f4356041b0d1.png)